src/System.Net.ServicePoint/tests/ServicePointManagerTest.cs

```diff
@@ -114,6 +114,7 @@ public static void ReusePort_Roundtrips()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Desktop default SecurityProtocol to Ssl3; explicitly changed to SystemDefault for core.")]
```

Comment:
>Will we want this test to run on .NET Framework when the tests run on 4.7?

Yes.  We will need to figure out how to do runtime detection of .NET Framework  version in these tests to validate against the changed default value.